client/src/apis/chessClient.sockets.js

```diff
@@ -0,0 +1,31 @@
+import IO from 'socket.io-client';
```

Comment:
is it a convention to name these files ending with .socket.js?
if not we can just have something like gameSocket.js or something similar.
As client in chessClient is implicit, from the file path client/....

Reply:
changed filename to chessSocket.js